flixel/ui/FlxBar.hx

```diff
@@ -198,9 +198,9 @@ class FlxBar extends FlxSprite
 	{
 		positionOffset = FlxDestroyUtil.put(positionOffset);
 
-		if (FlxG.renderBlit)
+		if (FlxG.renderTile)
 		{
-			_frontFrame = null;
+			frontFrames = null;
```

Comment:
Can you shed some light on this change? was `renderBlit` a mistake, or was the comment above wrong? why frontFrames instead of _frontFrames?

Reply:
`_frontFrame` and `_filledFlxRect` are only used in tile render mode, but the current code mistakenly tries to destroy these when the game is using blit render mode, and vice versa for the variables only used in blit mode. Simply changing line 201 to `renderTile` fixes it.

I changed line 203 to `frontFrames` as that value is left without being set to null in the current code, and `set_frontFrames` will also null out `_frontFrame`.